go/binlog/binlog_entry.go

```diff
@@ -1,27 +1,28 @@
 /*
-   Copyright 2016 GitHub Inc.
+   Copyright 2021 GitHub Inc.
 	 See https://github.com/github/gh-ost/blob/master/LICENSE
 */
 
 package binlog
 
 import (
 	"fmt"
+
 	"github.com/github/gh-ost/go/mysql"
+
+	gomysql "github.com/siddontang/go-mysql/mysql"
 )
 
 // BinlogEntry describes an entry in the binary log
 type BinlogEntry struct {
 	Coordinates mysql.BinlogCoordinates
-	EndLogPos   uint64
-
-	DmlEvent *BinlogDMLEvent
+	DmlEvent    *BinlogDMLEvent
```

Comment:
I see `EndLogPos` is removed. Does that mean the new `GTID` support completely removes support for non-GTID? Doesn't look like that from the rest of the code, but still curious why `EndLogPos` is removed.

Reply:
@shlomi-noach I wasn't able to find any instances where `EndLogPos` was used in the code so I removed it, but it's not required. In my short testing file-based binlogs are still working without this field

Reply:
sounds good. Thanks for explaining!